inspire_dojson/api.py

```diff
@@ -47,6 +48,8 @@
 from .jobs import jobs
 from .journals import journals
 
+RE_INVALID_CHARS_FOR_XML = re.compile(u'[^\u0020-\uD7FF\u0009\u000A\u000D\uE000-\uFFFD\U00010000-\U0010FFFF]+')
```

Comment:
I should probably credit the answer on StackOverflow for this particular regular expression...